device/workflow.go

```diff
@@ -9,17 +9,46 @@ import (
 
 // BadRepoError is an error message
 const BadRepoError = "Bad repository "
+const CustomFlash = "Custom"
```

Comment:
__[golint]__
* [golint] exported const CustomFlash should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->